### PR TITLE
Simplify boolean conditions

### PR DESCRIPTION
**Pull request by rhumbertgz**

### CODE REVIEW

File: appdaemon/plugins/mqtt/mqttplugin.py [hunk 1]
```diff
@@ -401,7 +401,7 @@ def start_mqtt_service(self, first_time):
                     auth.update({"keyfile" : self.mqtt_client_tls_client_key})
                     set_tls = True
                    
-                if set_tls == True:                    
+                if set_tls:
```
Comment (acockburn) [inline appdaemon/plugins/mqtt/mqttplugin.py]: For this I prefer `if set_tls is True:`

File: appdaemon/scheduler.py [hunk 2]
```diff
@@ -381,7 +381,7 @@ def get_next_entries(self):
                 if self.realtime is True:
                     self.now = now
                 else:
-                    if result == True:
+                    if result:
```
Comment (acockburn) [inline appdaemon/scheduler.py]: and this

File: appdaemon/plugins/mqtt/mqttplugin.py (hunk 1 above)
Comment (rhumbertgz) [reply appdaemon/plugins/mqtt/mqttplugin.py]: done!

File: appdaemon/scheduler.py (hunk 2 above)
Comment (rhumbertgz) [reply appdaemon/scheduler.py]: done!

**Pull request merged**